pkg/types/upgrades.go

```diff
@@ -26,6 +26,9 @@ func NewChangeDiff(componentReports ...*ComponentChangeDiff) *ChangeDiff {
 func (c *ChangeDiff) Append(changeDiffs ...*ChangeDiff) {
 	for _, diff := range changeDiffs {
 		if diff != nil {
+			if c == nil {
```

Comment:
How is this possible if `c` is a member method?

Reply:
Nvm after seeing https://stackoverflow.com/questions/42238624/calling-a-method-on-a-nil-struct-pointer-doesnt-panic-why-not